recipes/dbus/1.x.x/conanfile.py

```diff
@@ -0,0 +1,110 @@
+import os
+import shutil
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class DbusConan(ConanFile):
+    name = "dbus"
+    license = ("AFL-2.1", "GPL-2.0-or-later")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.freedesktop.org/wiki/Software/dbus"
+    description = "D-Bus is a simple system for interprocess communication and coordination."
+    topics = ("conan", "dbus")
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "fPIC": [True, False],
+        "with_x11": [True, False],
+        "with_glib": [True, False],
+        "enable_assert": [True, False],
+        "enable_checks": [True, False],
```

Comment:
Are these options needed?
These look like development options.
I would not make these options available.
Also, disable assertions and checks by default.

Reply:
By default it's enabled: 

```
option (DBUS_DISABLE_ASSERT "Disable assertion checking" OFF)
...
if (UNIX AND NOT DBUS_DISABLE_ASSERT)
	# required for backtrace
	SET(CMAKE_C_FLAGS_DEBUG   "${CMAKE_C_FLAGS_DEBUG}   -Wl,--export-dynamic")
	SET(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -Wl,--export-dynamic")
	add_definitions(-DDBUS_BUILT_R_DYNAMIC)
endif (UNIX AND NOT DBUS_DISABLE_ASSERT)
```
I also prefer it disabled, but it doesn't harm.

Reply:
It's perhaps safer. It's always nice to have extra checks for IPC.